### PR TITLE
Updates mkl_aarch64 build to use oneDNN v2.6.

### PR DESCRIPTION
**Pull request by nSircombe**

### CODE REVIEW

**Comment by nSircombe** [comment]
Hi @penpornk - I notice there's no r2.9 branch yet, is there any chance we can get this in before the cut?

File: tensorflow/workspace2.bzl [hunk 1]
```diff
@@ -184,9 +184,9 @@ def _tf_repositories():
         name = "mkl_dnn_acl_compatible",
         build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
         patch_file = ["//third_party/mkl_dnn:onednn_acl.patch"],
```
Comment (penpornk) [inline tensorflow/workspace2.bzl:186]: Do you still need the patch file? I guess you do since we are updating from 2.6-rc to 2.6?

**Review by penpornk — APPROVED** [review]
I think we are waiting to see the nightly test results and then cut at a commit from yesterday. But we can try to get this in anyway.

File: tensorflow/workspace2.bzl (hunk 1 above)
Comment (nSircombe) [reply tensorflow/workspace2.bzl:186]: Yes, for now it's still needed.
This patch was introduced with #55404 as part of the fix for accuracy issues seen in training workloads - it should be possible to remove the patch for a future oneDNN release, but at present the fix is needed for both the RC and Release versions of oneDNN.

**Comment by nSircombe** [comment]
> I think we are waiting to see the nightly test results and then cut at a commit from yesterday. But we can try to get this in anyway.

Righto, thanks @penpornk.

**Pull request merged**